src/utils.js

```diff
@@ -158,3 +158,16 @@ export function sorter ($sort) {
     return comparator;
   };
 }
+
+export function makeUrl (path, app = {}) {
+  const get = typeof app.get === 'function' ? app.get : () => {};
```

Comment:
Can this just take a normal HTTP request object instead? Creating a URL only really makes sense in the context of Express/HTTP so we should have access to it right?